<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FollowerInfo</class>
 <widget class="QDialog" name="FollowerInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1451</width>
    <height>854</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>51</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>用户名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>关注人数:</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>571</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>40</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>666</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>666</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
   <property name="text">
    <string>666</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>750</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
font: 25 12pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);

width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>查看问题</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>167</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
font: 25 12pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);

width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>查看详情</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>800</y>
     <width>93</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
font: 25 12pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);

width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>220</y>
     <width>751</width>
     <height>401</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 14pt &quot;华文行楷&quot;;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>750</y>
     <width>93</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
font: 25 12pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);

width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>查看回复</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>640</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
font: 25 12pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);

width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>640</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
font: 25 12pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);

width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>650</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
font: 25 12pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);

width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>上一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>650</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
font: 25 12pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);

width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>下一页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>150</y>
     <width>101</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
border-radius:8px;
font: 25 12pt &quot;华文行楷&quot;;
background-color: rgb(144,238,144);

width:71px;
 height:22px;</string>
   </property>
   <property name="text">
    <string>关注</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
